syncthing_gtk/wizard.py

```diff
@@ -200,6 +200,10 @@ def prepare(self):
 		""" Sets page as complete by default """
 		self.parent.set_page_complete(self, True)
 
+	def init_page(self):
```

Comment:
Other functions call this function, which doesn't exist on the master class, but is supposed to be implemented in subclasses.  The best solution is to make it exist, but raise an error like this.